### PR TITLE
Fix error with highlighting on code-blocks [13179]

### PR DESCRIPTION
**Pull request by richiware**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -344,7 +344,7 @@ def configure_doxyfile(
 # show_authors = False
 
 # The name of the Pygments (syntax highlighting) style to use.
-pygments_style = 'sphinx'
+# pygments_style = 'sphinx'
```
Comment (JLBuenoLopez-eProsima) [inline docs/conf.py:347]: I think this should not be removed so we can keep the greenish background instead of the gray one in the code snippets.

**Review by JLBuenoLopez-eProsima — CHANGES_REQUESTED** [review]
LGTM, though I would enable the `sphinx` pygment style to keep the same style we had before.

File: docs/conf.py (hunk 1 above)
Comment (richiware) [reply docs/conf.py:347]: I know but I think gray background color matches better with the rest of the colors in the documentation. @EduPonz  and @MiguelCompany what do you think?

Comment (EduPonz) [reply docs/conf.py:347]: I also think the greenish one stands out more than the grey

**Review by EduPonz — CHANGES_REQUESTED** [review]

File: docs/conf.py (hunk 1 above)
Comment (richiware) [reply docs/conf.py]: done

**Review by JLBuenoLopez-eProsima — APPROVED** [review]
LGTM

**Review by EduPonz — APPROVED** [review]

**Pull request merged**